Python Match Simulator/RobotQuality.py

```diff
@@ -136,5 +182,51 @@
             "Cycle_StdDev": 3.5,
             "Reliability": 50
         }
+    },
+    "DEAD": {

```

Comment:
Set all reliabilities to 0

Reply:
We also need to resolve the issue that a "dead" robot will still be able to reduce the quantities of paintings statues. I believe this can be resolved in the task selection method.

Reply:
Ok I will take a crack at it as part of this PR. Will push another commit here in a bit